spec/source/locales-currencies-tz.html

```diff
@@ -0,0 +1,163 @@
+<emu-clause id="locales-currencies-tz">
```

Comment:
@rwaldron no so sure about the id, we can either go with something smaller but more cryptic, or the full blow of the h1.